### DISCUSSION TITLE
Someone ring the bell / no, just walk

### DISCUSSION THREAD

**Discussion by rereone** [dgreif/ring#621] [General]
Help Please : 

Hello,

For the past few updates, my Ring doorbell has been sending me "ringing" notifications when someone walks past.

I can't seem to change that. Do you have an idea?

Especially since I have a HomeMini which "ding" every time.

Thank you.

**Comment by dgreif** [comment]
Do you have the `sendDoorbellMotionNotificationsToTv` option enabled?  If so, the notifications are behaving as intended. tvOS will only show a live stream for doorbell presses, so the plugin has to simulate a doorbell press when motion is detected

**Comment by rereone** [comment]
Thanks. It's correct. 

A new version with the two ?

**Reply by dgreif** [reply]
I'm sorry, I'm not sure what you are asking.  Would you mind clarifying?
